### ISSUE TITLE
Accessibility:  Related posts lists does not have list item tags

### ISSUE THREAD

**Issue by evelynhathaway** [unicorn-utterances/unicorn-utterances#737]
On the uwu branch (PR https://github.com/unicorn-utterances/unicorn-utterances/pull/497), the "Related posts" list on the right side should have `<li>` elements between the `<ul>` and the `<a>`.
